tests/whisper_protocol/text_secure_v3.rs

```diff
@@ -54,6 +58,58 @@ pub struct MessageKey{
     mac_key : [u8;32],
     iv : [u8;16],
 }
+#[derive(Clone)]
+#[derive(Debug)]
+#[derive(PartialEq)]
+pub struct TransportPacket {
+    version: u8,
+    payload : WhisperMessage,
+    mac : Option<Vec<u8>>,
```

Comment:
can this be changes to Option<[u8;TRUNCATED_MAC_LEN]> without a terrible amount of trouble?